protocol.html

```diff
@@ -609,14 +614,14 @@ <h3 property="schema:name">Notification Channel</h3>
                   <section id="notification-channel-types" inlist="" rel="schema:hasPart" resource="#notification-channel-types">
                     <h4 property="schema:name">Notification Channel Types</h4>
                     <div datatype="rdf:HTML" property="schema:description">
-                      <p>A notification channel type is a specific communication protocol that governs how classes of products can establish a connection to send and receive messages. A notification channel type also specifies the data model and format of the messages that are exchanged between products. A notification channel type is identified by a IRI for use with both <a href="#discovery">discovery</a> and the establishment of notification channels.</p>
+                      <p>A notification channel type is a specific communication protocol that governs how classes of products can establish a connection to send and receive messages. A notification channel type is identified by a IRI used to both <a href="#discovery">discover</a> and establish the <a href="#notification-channel">notification channels</a>.</p>
```

Comment:
I think the following information is significant enough to be mentioned here:

>specifies the data model and format of the messages that are exchanged between products

```suggestion
                      <p>A notification channel type is a specific communication protocol that governs how classes of products can establish a connection to send and receive messages. A notification channel type is identified by a IRI used to both <a href="#discovery">discover</a> and establish the <a href="#notification-channel">notification channels</a>. A notification channel type also specifies the data model and format of the messages that are exchanged between products.</p>
```

Reply:
This was one of the places where I should have discussed first. But I do not understand how the channel type specifies a data model. I thought the notification channel data model specifies the data model for the message. Similarly, isn't the format JSON-LD? I have not seen one type specifying these things, the only extra thing it can specify afaict is features. What am I missing here?

Reply:
I get you but I ask that we rethink the language and make things more explicit. What confused me was the use of the word "message" which I confused with "notification message". Whereas what I understand you are trying to say is that message between Client, Resource Server and Subscription Server can change based on the notification channel type.

Re: Accept feature and message format, I think of it as orthogonal to notification channel type. I don't think we want a new channel type being defined just because you want to send the message in a different data model and format. Hence, my original objection and removal of the statement.

Reply:
If WebSocketChannel2025 for any reason needs to require Turtle as the primary format and throw in additional core properties and possibly even features that are only meaningful/useful for WebSocketChannel2025, that'd would be allowed.

Besides those kind of scenarios, I would agree that there probably wouldn't be multiple notification channel types where their only difference would be the the data format. I can't see why WebSocketChannel2023 (sticking to JSON-LD default) and where WebSocketChannel2023-Turtle needs to be exist. This is a bit too hypothetical.

I've updated the language to indicate that it is a possibility.